editor/tests/org.fusesource.ide.camel.editor.tests.integration/src/main/java/org/fusesource/ide/camel/editor/integration/globalconfiguration/dataformat/wizards/NewDataFormatWizardIT.java

```diff
@@ -63,8 +63,7 @@
 	@Parameters(name = "{0} - {1}")
 	public static Collection<Object[]> data() {
 		List<String> supportedCamelVersions = CamelModelFactory.getSupportedCamelVersions();
-		// TODO: update when a new version is available
-		Assertions.assertThat(supportedCamelVersions).hasSize(2);
+		Assertions.assertThat(supportedCamelVersions.size()>0).isTrue();

```

Comment:
I know.
I have no better solution yet.
On the other hand, we need to add the bundles in manifest.MF I think to make it break. Unless we add the bundles for all tests everytime in a Tycho configuration. Seems a bit complex to handle in this PR.
I let you choose what to do in this PR (just checking not empty or th eexact number)

if check for nto empty better to use assertThat(supportedCamelVersions).isNotEmpty()